src/main/java/no/digipost/signature/client/direct/DirectClient.java

```diff
@@ -144,5 +145,8 @@ public InputStream getPAdES(PAdESReference pAdESReference) {
         return client.getSignedDocumentStream(pAdESReference.getpAdESUrl());
     }
 
+    public void deleteDocuments(DeleteDocumentsUrl deleteDocumentsUrl) {
```

Comment:
Vi burde nok ha et par setninger med dokumentasjon her. Bør for eksempel beskrive at bare ferdigsignerte oppdrag kan slettes, og at avsender må vente 24 timer etter signering.